src/utils/DragAndDrop.js

```diff
@@ -332,13 +336,16 @@ define(function (require, exports, module) {
          * or not a mime type we care about, reject it.
          */
         function rejectImport(item) {
-            if (item.size > byteLimit) {
-                return new Error(Strings.DND_MAX_FILE_SIZE_EXCEEDED);
-            }
+            var ext = FilerUtils.normalizeExtension(Path.extname(item.name), true);
```

Comment:
You don't need to call `FilerUtils.normalizeExtension()` here, since you already call it in `Content.isArchive()`.  Basically, we need it, but we only need it once.